slack/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+  - abhi-g
```

Comment:
remove @abhi-g ; he's working on other things.
No need to list me as an OWNER since I'm in the root and don't have additional context here.